units/src/amount.rs

```diff
@@ -0,0 +1,2354 @@
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
Why doesn't git mark this as moved file?

Reply:
Is it because we keep the `bitcoin/src/amount.rs` file? If it helps, during rebase, I diffed the new `amount.rs` file against the original to see I didn't make any mistakes.

Reply:
Ah, then why do we keep it?